tests/gui/conftest.py

```diff
@@ -0,0 +1,27 @@
+from streamlit.testing.v1 import AppTest
+from alphastats.load_data import load_data
+from pathlib import Path
+from alphastats import DataSet
+
+
+APP_FOLDER = Path(__file__).parent / Path("../../alphastats/gui/")
+TEST_INPUT_FILES = f"{APP_FOLDER}/../../testfiles"
+
+
+def print_session_state(apptest: AppTest):
+    for k, v in apptest.session_state.filtered_state.items():
+        print(
+            f"{k}:    {str(type(v))}   {str(v)[:20] if type(v) not in [int, list, str] else v}"
+        )
+
+
+def create_dataset_alphapept():
```

Comment:
by convention, conftest is at the root of the tests directory

Reply:
intended for helpers that are only used part of the tests